tests/cpp/test_remove_bcast_squeeze.cpp

```diff
@@ -0,0 +1,281 @@
+// clang-format off
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2023-present NVIDIA CORPORATION & AFFILIATES.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+// clang-format on
+#include <csrc/exceptions.h>
+#include <gmock/gmock-matchers.h>
+#include <gtest/gtest.h>
+
+#include <ir/all_nodes.h>
+#include <ir/utils.h>
+#include <ops/all_ops.h>
+#include <preseg_passes/optimization_pass.h>
+#include <preseg_passes/pre_segmenter.h>
+#include <tests/cpp/utils.h>
+#include <tests/cpp/validator.h>
+
+#include <torch/torch.h>
+
+namespace nvfuser {
+
+using RemoveBcastSqueezeTest = NVFuserTest;
+
+namespace {
+std::pair<bool, bool> hasBcastSqueeze(Fusion* fusion) {
```

Comment:
How about removing this helper in favor of https://github.com/NVIDIA/Fuser/blob/1e48b3752f8cfc976c83df5f31b6ddc2b12f6fc6/csrc/ir/utils.h#L601?

Reply:
changed to use `hasOpsOfType`